benchmarks/README.md

```diff
@@ -17,9 +55,106 @@ posting results compared against their base commit as comments.
+
+* [`Dylib`](https://google.github.io/iree/deployment-configurations/cpu-dylib/):
+  For CPU via dynamic library. Kernels contain CPU native instructions AOT
+  compiled using LLVM. This driver issues workload to the CPU in async
+  manner and supports multithreading.
+* [`Dylib-Sync`](https://google.github.io/iree/deployment-configurations/cpu-dylib/):
+  For CPU via dynamic library. Kernels contain contain CPU native instructions
+  AOT compiled using LLVM. This driver issues workload to the CPU in sync
+  manner.
+* [`VMVX`](https://github.com/google/iree/issues/5123):
+  For CPU via dynamic library. Kernels contain vector-level intrinsics that
+  are backed by fast implementations ([WIP](https://github.com/google/iree/issues/5819)).
+  This driver issues workload to the CPU in async manner and supports
+  multithreading.
+* [`Vulkan`](https://google.github.io/iree/deployment-configurations/gpu-vulkan/):
+  For GPU via Vulkan. Kernels contain SPIR-V. This driver issues workload to
+  the GPU via the Vulkan API.
+
+### Device name and target architecture
+
+This two fields are tightly coupled. They specify the device and hardware
+target for executing the benchmark.
+
+Right now there are two Android devices:
+
+* `Pixel-4`: Google Pixel 4 running Android 11. The SoC is
+  [Snapdragon 855](https://www.qualcomm.com/products/snapdragon-855-plus-and-860-mobile-platform),
+  with 1+3+4 ARMv8.2 CPU cores and Adreno 640 GPU.
+* `SM-G980F`: Samsung Galaxy S20 running Android 11. The SoC is
+  [Exynos 990](https://www.samsung.com/semiconductor/minisite/exynos/products/mobileprocessor/exynos-990/),
+  with 2+2+4 ARMv8.2 CPU cores and Mali G77 MP11 GPU.
+
+Therefore the target architectures are:
+
+* `CPU-CPU-ARMv8.2-A`: can benchmark all CPU-based IREE backends and drivers.
+* `GPU-Adreno-640`: can benchmark IREE Vulkan with Adreno target triples.
+* `GPU-Mali-G77`: can benchmark IREE Vulkan with Mali target triples.
+
+### Benchmark mode
+
+This field is to further specify the benchmark variant, given the same input
+model and target architecture. It controls important aspects like:
+
+* `*-core`: specifies the core flavor for CPU.
+* `*-thread`: specifies the number of threads for CPU.
+* `full-inference`: measures the latency for one full inference. Note that this
+  does not include the IREE system initialization time.
+* `kernel-execution`: measures only kernel execution latency for GPU. Note that
+  this is only possible for feedforward NN models that can be put into one
+  command buffer.
+
+`*-core` and `*-thread` together determines the `taskset` mask used for
+benchmarking IREE backends and drivers on CPU. For example,
+
+* `1-thread,big-core` would mean `taskset 80`.
+* `1-thread,little-core` would mean `taskset 08`.
+* `3-thread,big-core` would mean `taskset f0`.
+* `3-thread,little-core` would mean `taskset 0f`.
+
+## IREE developer guides
+
+The following sections are meant for developers working on IREE itself.
```

Comment:
That's what I intended for the README in general :P benchmark definitions come with documentation for how to edit those definitions and a brief overview of how they are used. Philosophy and interpretation docs I imagined going elsewhere.

Reply:
Yeah. I'm back and forth on where to put this. Not sure we want to put it on https://google.github.io/iree/ given this is quite internal development facing (although other interested folks will also check them so worth having documentation like this). In `docs/developers` we already have a few benchmarking related docs for  benchmarking flow, which is even lower level than this. So I figured right now it's better to get the content out instead of finding the perfect file/dir organization for it. So put it here. ;-P 

Splitted into a separate `dashboard.md` file now in f984896.

Reply:
f984896 is purely splitting the file. Comments were addressed in 53c0103.